### PR TITLE
OCPEDGE-1198: feat: adding timing information to the ibu and ibi steps

### PR DESCRIPTION
**Pull request by jeff-roche**

### CODE REVIEW

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.18-image-based-install pull-ci-openshift-kni-lifecycle-agent-release-4.17-ibu-e2e-flow

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.18-image-based-install pull-ci-openshift-kni-lifecycle-agent-release-4.17-ibu-e2e-flow

File: ci-operator/step-registry/openshift/image-based/install/openshift-image-based-install-commands.sh [hunk 1]
```diff
@@ -56,15 +56,31 @@ podman run -v ./tmp:/tmp:Z --user root:root --rm --entrypoint='["/bin/sh","-c"]'
...
+SECONDS=0
 make imagebasedconfig.iso ibi-attach-config.iso
+t_ibi_config=\$SECONDS
 
 echo "Rebooting the cluster"
+SECONDS=0
 make ibi-reboot wait-for-ibi
+t_ibi_config_reboot=\$SECONDS
 
+echo "IBI Times:"
```
Comment (suleymanakbas91) [inline ci-operator/step-registry/openshift/image-based/install/openshift-image-based-install-commands.sh:79]: Could you also add the time unit to the output?

**Review by suleymanakbas91 — CHANGES_REQUESTED** [review]

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.18-image-based-install pull-ci-openshift-kni-lifecycle-agent-release-4.17-ibu-e2e-flow

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.18-image-based-install pull-ci-openshift-kni-lifecycle-agent-release-4.17-ibu-e2e-flow

**Pull request merged**